### PR TITLE
Raghav - Amazon Web Implementation Assignment

### PR DESCRIPTION
**Pull request by raghavgarg1996**
Please review my assignment

### CODE REVIEW

File: src/test/java/com/znsio/sample/e2e/businessLayer/amazon/iphoneDetailPageBL.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import com.context.TestExecutionContext;
+import com.znsio.e2e.entities.Platform;
+import com.znsio.e2e.runner.Runner;
+import com.znsio.sample.e2e.businessLayer.jiomeet.LandingBL;
+import com.znsio.sample.e2e.entities.SAMPLE_TEST_CONTEXT;
+import com.znsio.sample.e2e.screen.amazon.IphoneDetailPageScreen;
+import org.apache.log4j.Logger;
+import org.assertj.core.api.SoftAssertions;
+
+public class iphoneDetailPageBL {
```
Comment (poojamanna18) [inline src/test/java/com/znsio/sample/e2e/businessLayer/amazon/iphoneDetailPageBL.java:12]: name of class is improper

File: src/test/java/com/znsio/sample/e2e/businessLayer/amazon/iphoneDetailPageBL.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    private static final Logger LOGGER = Logger.getLogger(LandingBL.class.getName());
```
Comment (poojamanna18) [inline src/test/java/com/znsio/sample/e2e/businessLayer/amazon/iphoneDetailPageBL.java:14]: class name is not proper

File: src/test/java/com/znsio/sample/e2e/businessLayer/amazon/iphoneDetailPageBL.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        softly = Runner.getSoftAssertion(threadId);
+        this.currentUserPersona = SAMPLE_TEST_CONTEXT.ME;
+        this.currentPlatform = Runner.platform;
+    }
+
+    public iphoneDetailPageBL authenticateIphone13DetailPage() {
+        IphoneDetailPageScreen
+                .get()
+                .selectFirstIphone();
+        return this;
```
Comment (poojamanna18) [inline src/test/java/com/znsio/sample/e2e/businessLayer/amazon/iphoneDetailPageBL.java:44]: assertion missing

File: src/test/java/com/znsio/sample/e2e/businessLayer/amazon/iphoneDetailPageBL.java [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    }
+
+
+    public iphoneDetailPageBL addToCart() {
+        IphoneDetailPageScreen
+                .get()
+                .addIphoneToCart();
```
Comment (poojamanna18) [inline src/test/java/com/znsio/sample/e2e/businessLayer/amazon/iphoneDetailPageBL.java:51]: assertion missing

File: src/test/java/com/znsio/sample/e2e/entities/SAMPLE_TEST_CONTEXT.java [hunk 5]
```diff
@@ -1,11 +1,29 @@
...
+    private  final String username;
+    private  final String password;
+
+    public SAMPLE_TEST_CONTEXT(String userpersona, Platform onPlatform) {
+        this.username = "sdhcdh@gmail.com";
+        this.password = "fdvdfvdfvdv";
+    }
+
+    public static String getUsername() {
+        return "raghav.gargril291900@gmail.com";
```
Comment (poojamanna18) [inline src/test/java/com/znsio/sample/e2e/entities/SAMPLE_TEST_CONTEXT.java:22]: Context variable usage is incorrect need to discuss

File: src/test/java/com/znsio/sample/e2e/screen/amazon/IphoneSuccessfullyAddedToCartScreen.java [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+import com.znsio.e2e.tools.Driver;
+import com.znsio.e2e.tools.Visual;
+import com.znsio.sample.e2e.screen.web.amazon.IphoneSuccessfullyAddedToCartScreenWeb;
+import org.apache.commons.lang3.NotImplementedException;
+import org.apache.log4j.Logger;
+
+import static com.znsio.e2e.runner.Runner.fetchDriver;
+import static com.znsio.e2e.runner.Runner.fetchEyes;
+
+public abstract class IphoneSuccessfullyAddedToCartScreen {
```
Comment (poojamanna18) [inline src/test/java/com/znsio/sample/e2e/screen/amazon/IphoneSuccessfullyAddedToCartScreen.java:14]: name of class should be proper

File: testLogs/teswizSampleTestLog.log [hunk 7]
```diff
@@ -0,0 +1,14 @@
+2022-11-02 17:20:54,633 - com.znsio.e2e.steps.RunCukes -INFO - RunCukes constructor: ThreadId: 1
```
Comment (vishal07111998) [inline testLogs/teswizSampleTestLog.log:1]: Do we require this log file.

File: src/test/resources/testData.json [hunk 8]
```diff
@@ -7,6 +7,13 @@
       "password": "eotTesting@0",
       "firstName": "Eot",
```
Comment (vishal07111998) [inline src/test/resources/testData.json:8]: Can we remove the test data that is not required

File: src/test/resources/indigo/Indigo.feature [hunk 9]
```diff
@@ -0,0 +1,10 @@
+
+Feature: Goindigo flight results
+
+      #  TAG="@indigo and web" PLATFORM=web  ./gradlew run
```
Comment (vishal07111998) [inline src/test/resources/indigo/Indigo.feature]: remove the tags that are not used

**Pull request closed without merging**